### DISCUSSION TITLE
[Feature] OLED Darkmode for Immich client application

### DISCUSSION THREAD

**Discussion by temal32** [immich-app/immich#13698] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, if you install Immich client app in the Google Play Store or iOS App Store it has a grey dark mode background, but I would like to have a full OLED black background, so pixels which are not needed are actually fully off to save battery.

For those who don't know what I mean by "OLED dark mode":
Normal dark mode: Grey background, pixels behind the display are ON to show grey color = needs battery
OLED dark mode: Black background, pixels behind the display are OFF = doesn't use any battery

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
You can go to the App Settings > Prefernces > Turn off Coloful Interface. I believe it will give you the black background in dark mode

**Reply by temal32** [reply]
Nope, unfortunately it doesn't work, it's still grey background just in another scheme.

**Reply by adveritas** [reply]
This is not true and we still have no AMOLED theme.

**Comment by Fiery-Penguin** [comment]
I would love this feature as well, OLED backgrounds a also a nice contrast to the foreground elements, as well as slight battery saving

**Comment by tam481** [comment]
Adding my +1 for AMOLED (true black) support

**Comment by Rygi3l** [comment]
Please implement this theme

**Comment by adveritas** [comment]
This feature is not implemented anyway we could get it? It's a very nice QoL improvement.
